plugins/syntax/numeral.ml

```diff
@@ -78,7 +78,7 @@ let locate_int63 () =
   else None
 
 let has_type f ty =
-  let (sigma, env) = Pfedit.get_current_context () in
+  let (sigma, env) = Pfedit.get_current_context () in (* XXX *)
```

Comment:
@JasonGross why are we using the proof-local environment here? Is this correct?

Reply:
@JasonGross ping

Reply:
The original numeral notations PR (#156) by @roglo uses `Lemmas.get_current_context ()`, and the revised one (#496) by @letouzey uses `Pfedit.get_current_context ()`.  I have not modified this bit of code, only moved it from one file to another, and don't know the reasoning behind it.  I believe this function is only called when invoking the `Numeral Notation` vernacular / when adding new numeral notations.  I think it would be fine to use the global env, or to get the env from the vernacular command and thread it through to this function call.